spec/unit/provider/package/pkgng_spec.rb

```diff
@@ -140,9 +142,10 @@ def run_in_catalog(*resources)
     # but neither of them works because provider.latest returns nil. This
     # is logged as issue PUP-4382
     #
-    # it("should retrieve the correct version of the latest package") {
-    #  provider.latest.should_not nil
-    #}
+    it("should retrieve the correct version of the latest package") {
+      provider_class.prefetch( { installed_name => installed_resource })
+      expect(installed_resource.provider.latest).not_to be_nil
+    }
```

Comment:
I'd drop the entire comment block above.

Also, for consistency with the rest of the file, I'd drop the parens on the `it`, and use `do ... end` rather than `{ ... }`.  Not sure where it ended up inconsistent, but might as well clean it up while we're in the neighborhood.